README.md

```diff
@@ -1,9 +1,6 @@
-<p align="center">
-  <br>
-    <img width="400" src="./assets/awesome-solid.png" alt="logo of awesome-solid-js repository"/>
-  <br>
-  <br>
-</p>
+<div align="center">
+  <img width="500" height="350" src="assets/awesome-solid.svg" alt="Awesome Solid">
```

Comment:
Please let me refer/check the logo guidelines for awesome list before going ahead with this merge. I'll introduce the necessary enhancements.

For now, could you please remove the logo changes from this PR?

Reply:
> For now, could you please remove the logo changes from this PR?

I'll keep this PR as it is.
I can rebase your changes from head, or discard this PR if you choose to go with your own.

Reply:
I've updated the description with guidelines followed:
* Used [logo](https://github.com/sindresorhus/awesome/blob/main/media/logo.png) from awesome GitHub repo.
* Used orange color complementary to blue used in SolidJS logo. The color in SolidJS logo was found by using [Digital Color Meter](https://g.co/kgs/VY129F), and complementary color was found using [Color calculator](https://www.sessions.edu/color-calculator/).
* Placed SolidJS logo in the center, just like Node.js hexagon is kept in [awesome-node](https://github.com/sindresorhus/awesome-nodejs#readme).
* Used recommended font [Orbitron](https://fonts.google.com/specimen/Orbitron) for text.